packages/cli/fragments/fragment-clojurescript/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "{{package_name}}",
+  "private": true,
+  "version": "0.0.0",
+  "scripts": {
+    "assets:build": "cp ./src/style.css ./public/",
+    "preshadow:watch": "npm run assets:build",
+    "shadow:watch": "npm run shadow-cljs watch app test",
+    "preshadow:build": "npm run assets:build",
+    "shadow:build": "npm run shadow-cljs release app",
+    "preshadow:test": "npm run assets:build",
+    "shadow:test": "npm run shadow-cljs compile ci && npm run karma start karma.config.js --single-run",
```

Comment:
test related, remove this along with anything relates to testing in this file, depdencies or scripts